pyproject.toml

```diff
@@ -16,13 +16,14 @@ click = "^8.1.7"
 wandb = "^0.18.1"
 huggingface-hub = {extras = ["cli"], version = "^0.25.1"}
 torch = { version = "2.5.0+cu121", source = "custom_torch"}
+jsonnet = "^0.20.0"
```

Comment:
https://github.com/google/jsonnet/issues/912

Reply:
Ok might have to set a different derivative of jsonnet as the dependency

Reply:
can you try `pip install jsonnet-binary`

Reply:
Works now

Reply:
was the fix jsonnet-binary?

Reply:
the easiest solution might just be to not use jsonnet - the only reason we use it is because the third-party evaluation pipeline has some util functions that we call on which use jsonnet. We can just re-write those to not use jsonnet.